### PR TITLE
🧹 Consistent task naming

### PR DESCRIPTION
**Pull request by ravinagill15**
## In this PR:
- Updated task names from `lz:validate-safe-configs` and `lz:validate-rpcs` to `lz:healthcheck:validate:safe-configs` and `lz:healthcheck:validate:rpcs`, respectively.
- Updated references within tests

### CODE REVIEW

File: packages/devtools-evm-hardhat/src/constants/tasks.ts
Comment (janjakubnanista) [inline packages/devtools-evm-hardhat/src/constants/tasks.ts:15]: `pnpm changeset` since this package is being published. Better avoid publishing the current version and just add a `patch` changeset

Comment (ravinagill15) [reply packages/devtools-evm-hardhat/src/constants/tasks.ts:15]: Ah good point will do!

**Review by janjakubnanista — DISMISSED** [review]

**Comment by ravinagill15** [comment]
Will merge after `chore/APPS-84` is merged to main so that PR is easier to review

**Review by janjakubnanista — APPROVED** [review]

**Pull request merged**
